fonts.css

```diff
@@ -0,0 +1,20 @@
+@font-face {
+font-family:"Montserrat";
+src: url("fonts/montserrat-500-normal.woff") format("woff"),
+url("fonts/montserrat-500-normal.woff2") format("woff2");
+font-weight: 500;
+}
+
+@font-face {
+font-family:"Montserrat";
+src: url("fonts/montserrat-600-normal.woff") format("woff"),
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla-n/pullRequest?prid=5749910)